Paws4Me/Paws4Me/Display Single Pet Details/View Controllers/PetSingleDetailsViewController.swift

```diff
@@ -17,15 +18,10 @@ class PetSingleDetailsViewController: UIViewController {
     @IBOutlet weak private var saveSinglePetButton: UIButton!
 
     // MARK: - Vars/Lets
-
+    private let viewContext = (UIApplication.shared.delegate as? AppDelegate)?.persistentContainer.viewContext
```

Comment:
Code like this does not belong in a `ViewController` this `ViewController` needs a `repository` 😄

Reply:
Thank you 👍🏻